uber/models.py

```diff
@@ -904,6 +912,12 @@ def hotel_status(self):
         else:
             return 'Hotel nights: ' + hr.nights_display
 
+class WatchList(MagModel):
+    first_name = Column(UnicodeText)
+    last_name = Column(UnicodeText)
+    disabled = Column(Boolean, default=False)
```

Comment:
Sure, that makes sense.

Though depending on exactly how one uses the watch list, perhaps the disable button is there so you can re-enable it later... :P